src/main/java/de/tsystems/onsite/bookabooth/repository/BookingRepository.java

```diff
@@ -2,11 +2,18 @@
 
 import de.tsystems.onsite.bookabooth.domain.Booking;
 import org.springframework.data.jpa.repository.*;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
 
 /**
  * Spring Data JPA repository for the Booking entity.
  */
 @SuppressWarnings("unused")
 @Repository
-public interface BookingRepository extends JpaRepository<Booking, Long> {}
+public interface BookingRepository extends JpaRepository<Booking, Long> {
+    @Modifying
+    @Transactional
+    @Query("DELETE FROM Booking b WHERE b.company.id = :companyId")
```

Comment:
Selber SQL statements in Attributen schreiben, sollte man nur in Ausnahmefällen. Hier ist es besser JPA Query Methods (https://docs.spring.io/spring-data/jpa/reference/jpa/query-methods.html) zu nutzen. Du könntest einfach

`void deleteByCompanyId(Long companyId);`

schreiben. Die Annotationen @Modifying, @Transactional und @Query können weggelassen werden. Im BookingService würde man dann eine Methode deleteByCompany(Long companyId) einführen, welche wiederum die Methode aus dem Repository aufruft.

ABER: du brauchst für dein Ticket gar keine Buchungen löschen. ALSO: bitte entfernen.